.mci.yml

```diff
@@ -282,6 +282,9 @@ pre:
     params:
       script: |
         rm -rf "mongo-cxx-driver"
+        if [ ! -d /data/db ]; then
+            mkdir /data/db
```

Comment:
/data directory is automatically created by the hostinit script when mci provisions a new builder. we wouldn't want to create it ourselves anyway, because we need it to be mounted on ephemeral storage.